core/src/test/java/io/github/tcdl/msb/impl/RequesterImplTest.java

```diff
@@ -81,9 +77,20 @@ public void testPublishWaitForResponses() throws Exception {
     }
 
     @Test
+    public void testPublishWaitForResponsesAck() throws Exception {
+        RequesterImpl<RestPayload> requester = initRequesterForResponsesWith(1, 1000, 800, arg ->  fail());
```

Comment:
it does we create collector with expectation of one message, then we received this message (handleMessage() ) and verify end() event was not fired (arg ->  fail())